cypress/utils/capture.js

```diff
@@ -6,6 +6,7 @@ export const CAPTURE_APP_URL = "dhis-web-capture/#";
 export const Selectors = {
   NEW_EVENT_BUTTON: '[data-test="new-button-toggle"]',
   SAVE_BUTTON: '[data-test="main-button"]',
+  SAVE_TEI_BUTTON: '[data-test="dhis2-uicore-splitbutton-button"]',
```

Comment:
this selector seems too generic - is there no more specific data-test from Capture itself, rather than the generic one from the UI library? I imagine it's easy that the UI would have another button with some combinations, or in the future .. having a more specific selector for saving TEI would make the test more resilient